frontend/src/App.jsx

```diff
@@ -13,16 +17,24 @@ import { BasicTemplateComponent } from './components';
 {/* Importing the items in Pages directory */}
 import { BasicTemplatePage } from './pages';
 
+
 function App() {
+  
+
   return (
+
+   
+    
     <Routes>
+      <Route path= "/app/login" element={<Login/>}></Route>
```

Comment:
Can you change the route to just "/login"? The "/app" prefix is meant for the main features